### ISSUE TITLE
Time zone always in UTC

### ISSUE THREAD

**Issue by kesnol** [IzakMarais/reporter#43]
Hello.
Great work with your reporter. 

The pdf report shows the time as UTC. Is there anyway to show it in other time zone like CET?
I have already tried to change the dashboard configuration but the report always shows UTC time zone.

**Comment by IzakMarais** [comment]
I think this is hard coded currently. 

Would you typically like to set this per dashboard as a query parameter, or would per reporter instance as a command line flag be sufficient?

**Comment by kesnol** [comment]
Any of the options you propose is fine for me, maybe using a variable is more flexible. 
As the developer you decide which one you like the most or the one that is easiest to implement.

Thanks!

**Comment by thnurg** [comment]
Hi Izak,
Thanks for this superb software. I suggest that the reporter get the server's local time zone, if possible, and use that as the default. The user could then override that with a query parameter.

**Comment by IzakMarais** [comment]
@thnurg Thanks for the suggestion, sounds easy enough. So if the server is in UTC the behaviour stays unchanged. I will implement this when I have time, sounds like a quick and easy change.

**Comment by thnurg** [comment]
I just had a thought regarding daylight savings. Those of use who practice that needless software overhead every year will probably have had the change over the last few weeks, so I pulled a report for March 22nd. (We switched from GMT to BST on March 24th.) The report showed the time in GMT. If I generate one for today then it will show BST. Go is obviously handling the server's timezone correctly. I wondered if daylight savings might cause issues generating reports that occurred before the most recent change but am happy to report that's not the case.
